packages/nextjs/src/index.server.ts

```diff
@@ -79,6 +79,15 @@ export function init(options: NextjsOptions): void {
   const activeDomain = domain.active;
   domain.active = null;
 
+  if (typeof options.tunnel !== 'undefined') {
+    try {
+      new URL(options.tunnel);
```

Comment:
Throwing an error/crashing is generally not something the SDK should do. I suggest we simply unset the `tunnel` option in the case it is a relative path and log a warning message (regardless of whether `debug` is set to `true` or not - so we should use `console.warn` instead of `logger.error` and remove the `__DEBUG_BUILD__` guard).

Reply:
Ah, I had wondered about that (reporting/throwing errors), I saw that there wasn't any precedent, but wasn't sure how the project handled it. Speaking of style:
- By "unset `tunnel`, " do you mean set to `undefined`, or using `delete` ?
- I'm getting eslint warning for `console.warn`, I assume you want `logger.warn` ?

Reply:
> * By "unset `tunnel`, " do you mean set to `undefined`, or using `delete` ?

Either way is fine I guess. Let's just set it to `undefined` for now.

> * I'm getting eslint warning for `console.warn`, I assume you want `logger.warn` ?

Feel free to disable that rule for that line and use `console.warn`. `logger.warn` will only print when `debug: true` is set and we wanna log that message in any case. The eslint rule just exists for us so we don't accidentally forget to use the logger.